theano/gof/graph.py

```diff
@@ -822,7 +822,7 @@ def clone(i, o, copy_inputs=True):
     return [equiv[input] for input in i], [equiv[output] for output in o]
 
 
-def clone_get_equiv(inputs, outputs, copy_inputs_and_orphans=True, memo=None):
+def clone_get_equiv(inputs, outputs, copy_inputs=True, copy_orphans=True, memo=None):
```

Comment:
In this file, there is a function that call clone_get_equiv while passing a parameter. This commit will change current Theano behavior. I don't think we want that. So I would add this diff to this PR:
```
diff --git a/theano/gof/graph.py b/theano/gof/graph.py
index d0c7652..9340c29 100644
--- a/theano/gof/graph.py
+++ b/theano/gof/graph.py
@@ -810,7 +810,7 @@ def clone(i, o, copy_inputs=True):
     o : list
         Output Variables.
     copy_inputs : bool
-        If True, the inputs will be copied (defaults to True).
+        If True, the inputs and orphans will be copied (defaults to True).
 
     Returns
     -------
@@ -818,7 +818,7 @@ def clone(i, o, copy_inputs=True):
         The inputs and outputs of that copy.
 
     """
-    equiv = clone_get_equiv(i, o, copy_inputs)
+    equiv = clone_get_equiv(i, o, copy_inputs, copy_inputs)
     return [equiv[input] for input in i], [equiv[output] for output in o]
 ```

You will need to update the docstring bellow.